elementor/elementor
# feat: accordion "keep first item closed" option

## PR Checklist



## PR Type
What kind of change does this PR introduce?


## Summary

This PR can be summarized in the following changelog entry:

* adds a checkbox to the accordion widget to disable "open first item"

## Description
An explanation of what is done in this PR

* adding a checkbox to the widget (default `yes`)
* adding a data attribute when its off (no HTML changes in the default way)
* adding a return in the JS function to skip clicking the first item

![Screenshot_20230223_124652](https://user-images.githubusercontent.com/4334997/220897834-3552c8d1-8c52-4df4-8f23-90f856bd8fe3.png)


## Test instructions
This PR can be tested by following these steps:

* add a accordion widget
* disable "open first item"
* check the frontend if it is closed on load

## Quality assurance


Fixes #
